Makefile

```diff
@@ -113,7 +119,7 @@ install-helm:
 
 .PHONY: e2e-teardown
 e2e-teardown:
-	helm delete csi-driver-smb --namespace kube-system
+	sudo /usr/local/bin/helm delete csi-driver-smb --namespace kube-system --kubeconfig ~/.kube/config

```

Comment:
that's env config issue: `sudo /usr/local/bin/helm`, better not hardcode in Makefile

Reply:
Ok. Removed those in the Makefile